pyproject.toml

```diff
@@ -66,7 +66,7 @@ color_output = true
 profile = "black"
 
 [tool.ruff]
-required-version = "0.0.265"
+required-version = "0.0.267"
```

Comment:
`required-version` must be kept in sync with the `pre-commit` version so as to not break code when Ruff is updated.